readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -154,6 +154,8 @@ else:
 # Add External version warning banner to the external version documentation
 if '{{ version.type }}' == 'external':
     extensions.insert(1, "readthedocs_ext.external_version_warning")
+    readthedocs_vcs_url = '{{ version.vcs_url }}'
+    readthedocs_build_url = '{{ build_url }}'
```

Comment:
Do we want these in the `html_context`?

Reply:
I'm fine adding them there, we don't use them in our theme, but I guess this makes it easy to use in other themes

Reply:
I think this is fine for now, no need to extend html_context without needing.

Reply:
+1 on that, bc if we added there we'll need to keep it forever...